config/environments/production.rb

```diff
@@ -81,8 +81,8 @@
   # Send deprecation notices to registered listeners.
   config.active_support.deprecation = :notify
 
-  # Use default logging formatter so that PID and timestamp are not suppressed.
-  config.log_formatter = ::Logger::Formatter.new
+  # Use custom formatter to support better error handling.
+  config.log_formatter = HartFormatter.new
```

Comment:
Using the new formatter is part of the application, so it's not specific to an environment, but should always be available in all environments. The formatter should be set in a general configuration, e.g. `config/application.rb`

Reply:
Ah, I was searching for a shared application file, but didn't find it. So `config/application.rb` is it.